src/components/Organisms/MarketingPreferencesDS/_MarketingPrefsConfig.js

```diff
@@ -0,0 +1,144 @@
+import * as yup from 'yup';
+import 'yup-phone';
+import { merge } from 'lodash';
+
+const setInitialValues = overrideValues => {
+  const defaultValues = {
+    mp_email: '',
+    mp_mobile: '',
+    mp_phone: '',
+    mp_address1: '',
+    mp_address2: '',
+    mp_address3: '',
+    mp_town: '',
+    mp_country: '',
+    mp_postcode: '',
+    mp_permissionEmail: [],
+    mp_permissionPost: [],
+    mp_permissionPhone: [],
+    mp_permissionSMS: []
+  };
+
+  // Override with any custom options supplied
+  const updatedValues = merge(defaultValues, overrideValues);
+
+  return updatedValues;
+};
+
+const buildValidationSchema = overrideOptions => {
+  const defaultOptions = {
+    mp_permissionEmail: {
+      /**
+        * As per react-hook-form's validation requirements, sets the fields' required attribute
+        * for each checkbox option (Yes & No), a non-required field isn't rendered or included
+        * in the validation. As ERP *currently* needs values to formalise a user's opt-out,
```

Comment:
Minor point but I don't think this restriction is to do with the ERP, so would maybe just say 'back end'

Reply:
> > > @AndyEPhipps do you have a link to the designs for this?
> > 
> > 
> > Ah yeah, sorry- here you go:
> > https://www.figma.com/file/MtusIm9S0M6Tj76AWVGcYe/CR-BAU-MarketingPrefs?node-id=166%3A99
> > I'd done a bit of a collab updates on this with Ruby, so not 100% sure if that's been updated. I'm gonna show her the build now too
> 
> Don't seem to have permish for this but doesn't matter!

I'm making a few tweaks with Ruby, i'll get her official stamp of approval eventually  😉